db/migrate/20161011213434_acts_as_votable_migration.rb

```diff
@@ -0,0 +1,27 @@
+class ActsAsVotableMigration < ActiveRecord::Migration
+  def self.up
+    create_table :votes do |t|
+
+      t.references :votable, :polymorphic => true
+      t.references :voter, :polymorphic => true
+
+      t.boolean :vote_flag
+      t.string :vote_scope
+      t.integer :vote_weight
+
+      t.timestamps
+    end
+
+    if ActiveRecord::VERSION::MAJOR < 4
```

Comment:
Given that this is a Rails 4 project and ActiveRecord's versions match that of Rails, I don't think we need this if statement or its contents.